waku/node/v2/wakunode2.nim

```diff
@@ -192,6 +192,12 @@ proc start*(node: WakuNode, conf: WakuNodeConf) {.async.} =
   node.switch.mount(wakuRelayProto)
   wakuRelayProto.started = true
 
+  let storeProto = WakuStore.init()
+  node.switch.mount(storeProto)
+
+  # @TODO THIS DOES NOT WORK, HOW DO I GET ACCESS TO `WakuRelay`
```

Comment:
@oskarth how would I get access to wakuRelay? do I need to cast something to it?

Reply:
I don't know off the top of my head. What did you try? Did you look at Nimbus code base as example?

Reply:
Hm, seems like there was some confusing references in the code, my bad. I fixed a bunch of it here https://github.com/status-im/nim-waku/pull/112

This means you wouldn't use wakuRelayProto. You should be able to get access to it via node and switch, see Node API and RPC for examples of this being done from node object.

Reply:
https://github.com/status-im/nim-waku/pull/112/files#r478829389